wolox-books/src/app/services/user.service.ts

```diff
@@ -30,8 +31,12 @@ export class UserService {
     });
     
     const url = `${this.root_url}/sessions`;
+    let service = this;
 
     let response = this.http.post(url, { session: login }, { headers })
-      .subscribe(token => console.log(token));
+      .subscribe(function (token) {
+        service.localStorageService.setValue('token', (<any>token).access_token);
```

Comment:
instead of using `'token'` use the variable declared in the local storage service (line 7) to avoid having a typo by repeating the same text multiple times